job/server/src/main/java/alluxio/job/load/LoadDefinition.java

```diff
@@ -58,14 +58,15 @@ public LoadDefinition() {
 
   @Override
   public Map<WorkerInfo, ArrayList<LoadTask>> selectExecutors(LoadConfig config,
-      List<WorkerInfo> jobWorkerInfoList, JobMasterContext jobMasterContext) throws Exception {
+      List<WorkerInfo> jobWorkerInfoList, SelectExecutorsContext selectExecutorsContext)
```

Comment:
Consider naming the argument `context` since it is the only context used in this method.

Reply:
done